Input/InputFieldSpotlightDecorator.js

```diff
@@ -230,7 +236,7 @@ const InputSpotlightDecorator = hoc(defaultConfig, (config, Wrapped) => {
 						this.blur();
 						forwardBlur(ev, this.props);
 					} else {
-						this.focusDecorator(ev.currentTarget);
+						this.fromMouse = false;
```

Comment:
@sjro Let's also set the `ev.currentTarget` to the node since we are not sure if there are any edge cases.